thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -80,12 +84,29 @@ public static ThriftCallService of(Object implementation) {
      *                        the Apache Thrift compiler
      */
     public static ThriftCallService of(Map<String, ?> implementations) {
+        requireNonNull(implementations, "implementations");
+        final ImmutableMap<String, ? extends ImmutableList<?>> implementationMap =
+                implementations.entrySet().stream()
+                               .map(e -> Maps.immutableEntry(e.getKey(), ImmutableList.of(e.getValue())))
+                               .collect(toImmutableMap(Entry::getKey, Entry::getValue));
+        return new ThriftCallService(implementationMap);
+    }
+
+    /**
+     * Creates a new multiplexed {@link ThriftCallService} with the specified list service implementations.
+     *
+     * @param implementations a {@link Map} whose key is service name and value is the list of implementations
+     *                        of {@code *.Iface} or {@code *.AsyncIface} service interface generated by
+     *                        the Apache Thrift compiler
+     */
+    public static ThriftCallService ofMultiple(Map<String, ? extends Iterable<?>> implementations) {
```

Comment:
Ahhh, erasures! I guess we need to make a breaking change here. Let's remove the old `of(Map)` and rename `ofMultiple()` back to `of()`.